code/datums/quirks/negative_quirks/scarred_eye.dm

```diff
@@ -0,0 +1,63 @@
+/datum/quirk/item_quirk/scarred_eye
+	name = "Scarred Eye"
+	desc = "An accident in your past has cost you one of your eyes, but you spawn with an eyepatch. Yarr!"
```

Comment:
```suggestion
	desc = "An accident in your past has cost you one of your eyes, but you got a cool eyepatch. Yarr!"
```
Try to avoid OOC terms whenever possible